docs/Developers/Testing_Bears.rst

```diff
@@ -125,7 +125,8 @@ passed.
                 self.uut,
                 file,
                 [Result.from_values('TooManyLinesBear',
-                                    'Too many lines')],
+                                    'Too many lines',
+                                    'files')],
```

Comment:
Actually, the parameter required by `from_values()` method is `file` which is the absolute path to the related file.So I think in place of `files` it should be `filename`.

Reply:
Okay. I will correct these. Thanks for guidance.